packages/react-checkbox/etc/react-checkbox.api.md

```diff
@@ -4,19 +4,23 @@
 
 ```ts
 
-import { IIconProps } from '@fluentui/react-internal/lib/Icon';
+import { BaseSlots } from '@fluentui/react-compose';
```

Comment:
You should remove the react-internal dep from everything except tabs and toggle now. (Also consider whether there are any other deps such as react-hooks that can be removed.)

Reply:
`slider` too since it uses `Label` but I agree.